src/pages/SignUp/Components/SignUpAfter.js

```diff
@@ -0,0 +1,107 @@
+import React, { useEffect, useState } from 'react';
+import { useNavigate } from 'react-router-dom';
+import * as S from './SignUpAfter.styled';
+
+const SignUpAfter = () => {
+  const [buttonDisable, setButtonDisable] = useState(false);
+  const [userInfo, setUserInfo] = useState({
+    id: '',
+    name: '',
+    email: '',
+  });
+  const navigate = useNavigate();
+  const handleUserInfo = e => {
+    const { name, value } = e.target;
+    setUserInfo(before => ({ ...before, [name]: value }));
+  };
+  const handleSubmit = () => {
+    fetch('http://10.58.7.122:8000/users/kakao', {
+      method: 'POST',
+      body: JSON.stringify({
+        kakao_id: userInfo.id,
+        user_name: userInfo.name,
+        email: userInfo.email,
+      }),
+    })
+      .then(res => res.json)
+      .then(result =>
+        result.message === 'LOGIN_MESSAGE'
+          ? navigate('/sign-in')
+          : alert('회원가입 오류')
+      );
```

Comment:
통신도 확인해보셨나요?? 파싱이 제대로 안 됐을 것 같네요!
```suggestion
      .then(res => res.json())
      .then(result =>
        result.message === 'LOGIN_MESSAGE'
          ? navigate('/sign-in')
          : alert('회원가입 오류')
      );
```

Reply:
네!! 파싱이 제대로 안됐었네요ㅠㅠㅠ json() 추가로 만들었습니다!